src/renderer/util/fileSystem.js

```diff
@@ -160,6 +164,23 @@ export const uploadImage = async (pathname, image, preferences) => {
       })
   }
 
+  const uploadByCliScript = (filepath, name = null) => {
+    let isPath = true
+    if (typeof filepath !== 'string') {
+      isPath = false
+      const data = new Uint8Array(filepath)
+      filepath = path.join(tmpdir(), name || +new Date())
+      writeFileSync(filepath, data)
+    }
+    cp.execFile(cliScript, [filepath], (err, data) => {
+      !isPath && unlink(filepath)
```

Comment:
This function does a lot of things but change at least the line above to a if-condition for readability.